src/audio/code_8019AF00.c

```diff
@@ -4215,7 +4215,10 @@ void AudioSfx_LowerSfxSettingsReverb(Vec3f* pos, s8 isReverbLowered) {
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_8019F638.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_8019F780.s")
+void func_8019F780(Vec3f* arg0, u16 sfxId, f32 arg2) {
+    func_8019F5AC(arg2);
+    AudioSfx_PlaySfx(sfxId, arg0, 4, &sSfxSyncedFreq, &sSfxSyncedVolume, &gSfxDefaultReverb);
+}
```

Comment:
This function is decompiled and named in the audio sfx PR, so no need to decompile it here

Reply:
Oh wtf, I don't remember this at all lol